### PR TITLE
Fixed issues and tested the functionalities

### PR DESCRIPTION
**Pull request by clemthenem**
@ouranos I fixed the locales on this one
@alexnoox I refactored some part of the code and fixed some design issues.
Let me know if I need to change anything!
Cheers

### CODE REVIEW

File: src/app/components/star-rating/star-rating.less [hunk 1]
```diff
@@ -2,9 +2,9 @@ average-star-rating, star-rating {
   position: relative;
   display: block;
 
-  .rating {
+  ul.rating {
     margin: 0;
-    padding: 0;
+    padding: 0!important;
```
Comment (clemthenem) [inline src/app/components/star-rating/star-rating.less]: This is due to a list overriding this property. Not good practice but in that case we really don't want any external css playing with the ratings

File: src/app/index.less [hunk 2]
```diff
@@ -46,6 +46,18 @@ html, body {
...
+.bottom-buffer-1 { margin-bottom: 1*@bottom-buffer-height; }
+.bottom-buffer-2 { margin-bottom: 2*@bottom-buffer-height; }
+.bottom-buffer-3 { margin-bottom: 3*@bottom-buffer-height; }
+.bottom-buffer-4 { margin-bottom: 4*@bottom-buffer-height; }
+.bottom-buffer-5 { margin-bottom: 5*@bottom-buffer-height; }
+.bottom-buffer-6 { margin-bottom: 6*@bottom-buffer-height; }
+.bottom-buffer-7 { margin-bottom: 7*@bottom-buffer-height; }
+.bottom-buffer-8 { margin-bottom: 8*@bottom-buffer-height; }
+.bottom-buffer-9 { margin-bottom: 9*@bottom-buffer-height; }
+.bottom-buffer-10 { margin-bottom: 10*@bottom-buffer-height; }
```
Comment (clemthenem) [inline src/app/index.less]: Not sure if we need it all as I am only usin g the bottom-buffer-10

File: src/app/components/star-rating/star-rating.less (hunk 1 above)
Comment (alexnoox) [inline src/app/components/star-rating/star-rating.less]: Should be `padding: 0 !important;`, with a space between `0` and `!`

File: src/app/index.less (hunk 2 above)
Comment (alexnoox) [inline src/app/index.less]: It is tool classes, let's keep them all

File: src/app/stylesheets/variables.less [hunk 3]
```diff
@@ -356,10 +356,15 @@
 /*-----------------------------------------------------------------------*/
 /*                          Reviews and comments                         */
 /*-----------------------------------------------------------------------*/
+@no-review-bg-panel-color:       #e2e1db;
```
Comment (alexnoox) [inline src/app/stylesheets/variables.less]: Typo on `mno`

**Comment by clemthenem** [comment]
@alexnoox  I fixed the indentation. 
@ouranos ?

File: src/app/stylesheets/variables.less (hunk 3 above)
Comment (alexnoox) [inline src/app/stylesheets/variables.less]: Do not use a specific color here, reuse one of the colors in `theme.less`

File: src/app/stylesheets/variables.less [hunk 4]
```diff
@@ -356,10 +356,15 @@
...
+
 @comment-bg:                     @bg-main-color;
 @comment-text-color:             @text-strong-color;
 @comment-attributes-color:       @bg-inverse-color;
 
-@question-search-border:         1px solid #c5c5c5;
+@no-question-bg-panel-color:     #e2e1db;
```
Comment (alexnoox) [inline src/app/stylesheets/variables.less]: Same here

File: src/app/views/marketplace/marketplace-app.controller.coffee [hunk 5]
```diff
@@ -63,7 +63,7 @@ angular.module 'mnoEnterpriseAngular'
         if vm.isReviewingEnabled
           fetchReviews(app.id, vm.reviews.nbItems, 0)
           fetchQuestions(app.id, vm.questions.nbItems, 0)
-          vm.averageRating = vm.app.average_rating? && Math.round(parseFloat(vm.app.average_rating).toFixed(1))
+          vm.averageRating = Math.round(vm.app.average_rating * 10) / 10
```
Comment (alexnoox) [inline src/app/views/marketplace/marketplace-app.controller.coffee]: Should be `vm.averageRating = parseFloat(vm.app.average_rating).toFixed(1)`

File: src/app/views/marketplace/marketplace-app.html [hunk 6]
```diff
@@ -116,16 +116,16 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
           <ul>
             <li ng-repeat="plan in vm.pricing_plans">
               <b>{{plan.name}}</b>: {{plan.description}}<br>
-              <span ng-if="plan.price.value">{{plan.currency}} {{plan.price.value}} {{plan.factor}}</span>
-              <span ng-if="!plan.price.value">{{plan.currency}} {{plan.price}} {{plan.factor}}</span>
+              <span ng-if="plan.price.value != null">{{plan.currency}} {{plan.price.value}} {{plan.factor}}</span>
+              <span ng-if="plan.price.value == null">{{plan.currency}} {{plan.price}} {{plan.factor}}</span>
```
Comment (alexnoox) [inline src/app/views/marketplace/marketplace-app.html]: The previous solution was not working?

File: src/app/views/marketplace/marketplace-compare.controller.coffee [hunk 7]
```diff
@@ -23,7 +23,13 @@ angular.module 'mnoEnterpriseAngular'
       MnoeMarketplace.getApps().then(
         (response) ->
           response = response.plain()
-          vm.apps = response.apps
+          # Filter apps selected
+          vm.compared_apps = _.each(
+            _.filter(response.apps, (app)-> app.toCompare == true),
+            (app) ->  # Round average rating
+              app.average_rating = Math.round( app.average_rating * 10 ) / 10;
```
Comment (alexnoox) [inline src/app/views/marketplace/marketplace-compare.controller.coffee]: Wrong formula, see above.

Comment (alexnoox) [inline src/app/views/marketplace/marketplace-compare.controller.coffee]: Fix indentation

File: src/app/views/marketplace/marketplace-compare.controller.coffee [hunk 8]
```diff
@@ -23,7 +23,13 @@ angular.module 'mnoEnterpriseAngular'
       MnoeMarketplace.getApps().then(
         (response) ->
           response = response.plain()
-          vm.apps = response.apps
+          # Filter apps selected
+          vm.compared_apps = _.each(
```
Comment (alexnoox) [inline src/app/views/marketplace/marketplace-compare.controller.coffee]: Use camelCase: `vm.comparedApps`

File: src/app/views/marketplace/modals/create-review-modal.html [hunk 9]
```diff
@@ -1,11 +1,11 @@
 <form id="create-review-modal" name="form" novalidate>
   <div class="modal-header">
     <div class="close" type="button" ng-click="vm.modal.cancel()" >×</div>
-    <h3>{{'mno_enterprise.templates.dashboard.marketplace.show.review' | translate}}</h3>
+    <h3>{{'mno_enterprise.templates.dashboard.marketplace.show.review_title' | translate}}</h3>
   </div>
   <!-- Modal Body -->
   <div class="modal-body">
-    <div class="pull-right">
+    <div class="pull-left">
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.html]: Doesn't it stay left if you remove the class?

File: src/app/views/marketplace/modals/edit-review-modal.html [hunk 10]
```diff
@@ -5,7 +5,7 @@
   </div>
   <!-- Modal Body -->
   <div class="modal-body">
-    <div class="pull-right">
+    <div class="pull-left">
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/edit-review-modal.html]: Idem

**Comment by clemthenem** [comment]
Done @alexnoox

**Pull request merged**